packages/forms/frontend/sirius-components-forms/src/propertysections/ButtonPropertySection.tsx

```diff
@@ -55,6 +57,10 @@ const useStyle = makeStyles<Theme, ButtonStyleProps>((theme) => ({
   icon: {
     marginRight: ({ iconOnly }) => (iconOnly ? theme.spacing(0) : theme.spacing(2)),
   },
+  div: {
```

Comment:
Could you rename this to something more specific like `propertySection` please?

Reply:
Could you also rename every other new `div` rules in others  `**PropertySection.tsx` please?

Reply:
Done.